packages/template-retail-react-app/app/pages/checkout/partials/shipping-multi-address.jsx

```diff
@@ -255,6 +277,40 @@ const ShippingMultiAddress = ({
         )
     }
 
+    // Handle customer loading state
+    if (customerLoading) {
+        return (
+            <Center p={8} textAlign="center" color="gray.500">
+                <VStack spacing={4}>
+                    <Spinner size="lg" />
+                    <Text>
+                        {formatMessage({
+                            id: 'shipping_multi_address.loading_customer.message',
+                            defaultMessage: 'Loading customer information...'
+                        })}
+                    </Text>
+                </VStack>
+            </Center>
```

Comment:
why only checking for customerLoading? What happens of useProducts are still loading? Instead of trying to get it to render different flow, can we simplify by using Skeleton wrapped around the component until we have all the data (both customer and useProducts)?

Reply:
Ok .. I'll try to do this.

Reply:
done